ee/vellum_ee/workflows/display/tests/workflow_serialization/test_scheduled_trigger_serialization.py

```diff
@@ -32,27 +32,37 @@ def mock_metadata_json(trigger_class_name: str, trigger_id: UUID) -> Iterator[Tu
     with tempfile.TemporaryDirectory() as tmpdir:
         # Create metadata.json with trigger mapping
         metadata_path = os.path.join(tmpdir, "metadata.json")
-        trigger_module_path = f"{__name__}.{trigger_class_name}"
+
+        test_module_parts = __name__.split(".")
+        workflow_root_module = ".".join(test_module_parts[:-1])  # Remove last part
+
+        relative_module_path = "." + test_module_parts[-1]
 
         with open(metadata_path, "w") as f:
             json.dump(
                 {
                     "trigger_path_to_id_mapping": {
-                        trigger_module_path: str(trigger_id),
+                        # Use relative module path (just the module, not the class name)
+                        relative_module_path: str(trigger_id),
```

Comment:
[nit] in test fixtures, may be easier to have hard coded paths instead of using logic